hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def get_middle(str)
+  hf = str.length / 2
+  if str.length.even?
+    str[hf - 1] + str[hf]
```

Comment:
and if you'll choose the ternary operator, the range looks better here `str[(hf - 1)..hf]`